doc/news/changes.h

```diff
@@ -1,1314 +0,0 @@
-// ---------------------------------------------------------------------
-//
-// Copyright (C) 2014 - 2016 by the deal.II authors
-//
-// This file is part of the deal.II library.
-//
-// The deal.II library is free software; you can use it, redistribute
-// it, and/or modify it under the terms of the GNU Lesser General
-// Public License as published by the Free Software Foundation; either
-// version 2.1 of the License, or (at your option) any later version.
-// The full text of the license can be found in the file LICENSE at
-// the top level of the deal.II distribution.
-//
-// ---------------------------------------------------------------------
-
-/**
-@page changes_after_8_4_2 Changes after Version 8.4.2
```

Comment:
you delete whole lot of things, but there is no creating of the same amount of files for each entry. 
I would suggest to keep the `changes.h` untouched in THIS pr and make sure that the script creates exactly the same file (disregarding whitespace and alike). Once this is the case, one can delete the `changes.h` and add it to `.gitignore`.